src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/PropertyPage.cs

```diff
@@ -363,7 +363,7 @@ public void SetObjects(uint cObjects, object[] ppunk)
                 }
             }
 
-            OnSetObjects(false);
+            WaitForAsync(() => OnSetObjects(false));
```

Comment:
This should either be a call to `SetObjects` if we intend to wait on the result of use a discard if we do not

Reply:
Anyone know what we intended?  Feels like wait, but I'm not familiar with this at all.

Reply:
I guess Discard is the thing that doesn't change behavior...

Reply:
`.Forget()`?